src/components/Sections/SectionsTypes.ts

```diff
@@ -0,0 +1,57 @@
+export interface DeviceSettings {
+  detailedLine: boolean
+  grouped: boolean
+}
+
+export interface Section {
+  id: string
+  name: string
+  items: SectionItem[]
+  layout: {
+    originalName: string
+    createdByApp: string
+    mobile: DeviceSettings
+    desktop: DeviceSettings
+    order: number
+  }
+}
+
+export interface SectionsLayout {
+  sections: Section[]
+}
+
+export interface SectionViewProps {
```

Comment:
I'm not familiar with Typescript but props would it be more easy to colocate it in the same file as the component ?

Reply:
Yeah it's very subjective.. some people don't like seeing types in runtime code files so I try in general to not mix them. But in the end it is purely a subjective matter, we don't have guidelines on this subject. I'm open to suggestions

Reply:
Yes, you're right, very subjective. Thank you for making the point. Since there's no guideline, I'm fine with it.